.changeset/friendly-bananas-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-hierarchies-react": minor
+---
+
+Added `onHierarchyLimitExceeded` callback for tracking when hierarchy level exceeds the limit.
```

Comment:
Please mention where it was added. Ideally, an example should also be included.

Reply:
Updated.